src/main/java/de/samply/common/ldmclient/cql/LdmClientCql.java

```diff
@@ -158,6 +172,81 @@ private QueryResultStatistic createQueryResultStatistic(MeasureReport measureRep
     return queryResultStatistic;
   }
 
+  /**
+   * Create a sublist for an entity.
+   *
+   * @param location the measure location url
+   * @return location url for the sublist
+   * @throws LdmClientException exception which can be thrown while posting the query to the ldm
+   */
+  public String createSubList(String location) throws LdmClientException {
+    JsonObject parameter = loadParameterStub();
+    String uri = location + "/$evaluate-measure";
+    HttpPost httpPost = new HttpPost(uri);
+    httpPost.setHeader(HttpHeaders.CONTENT_TYPE, "application/fhir+json");
+    HttpEntity entity = new StringEntity(parameter.toString(), Consts.UTF_8);
+    httpPost.setEntity(entity);
+    try (CloseableHttpResponse response = getHttpClient().execute(httpPost)) {
+      int statusCode = response.getStatusLine().getStatusCode();
+      if (statusCode != HttpStatus.SC_CREATED) {
+        logger.error(String.format("Sublist not created. Status code: %d, Response: %s",
+            statusCode, EntityUtils.toString(response.getEntity(), Consts.UTF_8)));
+        throw new LdmClientException("Request not created. Received status code " + statusCode);
+      }
+      Header locationHeader = response.getFirstHeader("Location");
+      if (locationHeader == null) {
+        throw new LdmClientException("Location header is missing");
+      }
+      String subListUrl = resourceLocation(locationHeader.getValue());
+      if (subListUrl.equals("")) {
+        throw new LdmClientException("Location header is empty");
+      }
+      return getSubListLocation(subListUrl);
+    } catch (IOException e) {
+      throw new LdmClientException(e);
+    }
+  }
+
+  private String getSubListLocation(String measureLocation) throws LdmClientException {
+    HttpGet httpGet = new HttpGet(measureLocation);
```

Comment:
An Accept header of `application/fhir+json` is missing here. It's not guaranteed that even Blaze will return JSOn by default.